tests/dummy/app/templates/load-previous.hbs

```diff
@@ -0,0 +1,23 @@
+<h2 id="posts-title">Listing Posts</h2>
+
+<div class="posts">
```

Comment:
Need to add a test for this route.  Perhaps need a config to scroll b/c `#ember-testing` is the `viewport` when we don't have a scrollable container.